apps/backoffice-v2/src/pages/Statistics/components/PortfolioRiskStatistics/hooks/usePortfolioRiskStatisticsLogic/usePortfolioRiskStatisticsLogic.tsx

```diff
@@ -44,27 +41,27 @@ export const usePortfolioRiskStatisticsLogic = ({
   }, [filteredRiskIndicators]);
   const filters = [
     {
-      name: 'Merchant Monitoring',
-      description: 'Risk Risk levels of all merchant monitoring reports.',
+      name: 'All Merchant Monitoring by Risk',
+      description: 'Risk levels of all merchant monitoring reports',
       entityPlural: 'Reports',
       riskLevels: {
-        low: reports.all.low,
-        medium: reports.all.medium,
-        high: reports.all.high,
-        critical: reports.all.critical,
-      },
-    },
-    {
-      name: 'Merchant Onboarding',
-      description: 'Risk levels of all active onboarding cases.',
-      entityPlural: 'Cases',
-      riskLevels: {
-        low: reports.inProgress.low,
-        medium: reports.inProgress.medium,
-        high: reports.inProgress.high,
-        critical: reports.inProgress.critical,
+        low: reportsRisks?.all?.low,
+        medium: reportsRisks?.all?.medium,
+        high: reportsRisks?.all?.high,
+        critical: reportsRisks?.all?.critical,
       },
     },
+    // {
+    //   name: 'Approved Merchant Onboarding',
+    //   description: 'Risk levels of all approved merchant onboarding reports',
+    //   entityPlural: 'Cases',
+    //   riskLevels: {
+    //     low: reportsRisks?.approved?.low,
+    //     medium: reportsRisks?.approved?.medium,
+    //     high: reportsRisks?.approved?.high,
+    //     critical: reportsRisks?.approved?.critical,
+    //   },
+    // },
```

Comment:
Remove commented out code. I'd keep it if it was something to remember or complex.